app/javascript/components/pill.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+class Pill extends React.Component {
+  render(){
+    const { className, url, text } = this.props;
+    const StylingClassName = `component-pill ${className}`;
```

Comment:
@aSadhankar looks good I’d just lower case StylingClassName to stylingClassName as caps is usually reserved for class names like component names. 

The other thing you could do, is not destructure class name from the props and have this:

```
  const className = `component-pill ${this.props.className`;
```

Reply:
Yeah not sure why I upper cased that! Made those changes 👍